lib/java/com/google/android/material/bottomnavigation/res/values/styles.xml

```diff
@@ -28,6 +28,7 @@
     <item name="itemPaddingBottom">@dimen/design_bottom_navigation_label_padding</item>
     <item name="itemActiveIndicatorStyle">@null</item>
     <item name="android:minHeight">@dimen/design_bottom_navigation_height</item>
+    <item name="android:fitsSystemWindows">true</item>
```

Comment:
Hmm, could you please clarify, is this a breaking change because a new attribute has been added, or because it's precisely the `android:fitsSystemWindows` attribute that has been added? Why is this change breaking?
I've made it by analogy with c66633b, do I understand correctly, that change was also breaking?

Reply:
I mean this will change every existing usage of bottom navigation view from android:fitsSystemWindows=false to android:fitsSystemWindows=true, right?

But I think you raised a good point. I'll ask the author of the change you refer to to confirm this. : )